api/python/tests/data/quilt.yml

```diff
@@ -0,0 +1,15 @@
+version: 1.0
+registries:
+  s3://some-bucket:
+    packages:
+      - asah/gpt3:latest
+      - akarve/lmnb1:c698234
+  s3://another-bucket:
+    packages:
+      - asah/gpt3
+      - akarve/lmnb1:c698234
+      - akarve/lmnb3/sub/path:c698234
```

Comment:
@akarve @cosmic-byte 
Intra-package paths (`sub/path` here) are not in spec and I'm not sure if they were requested/needed.

Reply:
We definitely need and want these. Greg please update the spec.

Reply:
I've updated the spec